app/oxide-rot-1/app.toml

```diff
@@ -114,6 +114,18 @@ in_cfg = [
     { pin = { port = 0, pin = 9 }, alt = 3 },
     { pin = { port = 0, pin = 8 }, alt = 0, mode = "pulldown" },
 ]
+# Connected = MOSI on, MISO off, SCK on
+connected_cfg = [
+    { pin = { port = 0, pin = 8 }, alt = 3 },
+    { pin = { port = 0, pin = 9 }, alt = 0, mode = "pulldown" },
+    { pin = { port = 0, pin = 7 }, alt = 3 },
+]
+# Connected = MOSI off, MISO off, SCK off
+disconnected_cfg = [
+    { pin = { port = 0, pin = 8 }, alt = 0, mode = "pulldown" },
+    { pin = { port = 0, pin = 9 }, alt = 0, mode = "pulldown" },
+    { pin = { port = 0, pin = 7 }, alt = 0, mode = "pulldown" },
+]
```

Comment:
Was about to ask whether this could be defined in the `base.toml` but then i realized...we don't have one for the RoT! Carry on!